src/test/outmdsd/testlogitem.cc

```diff
@@ -39,9 +40,14 @@ BOOST_AUTO_TEST_CASE(Test_LogItem_CacheTime)
         DjsonLogItem item(source, schemaAndData);
         item.Touch();
         int nexpectedMS = 20;
-        usleep(nexpectedMS*1000);
-        auto actualMS = item.GetLastTouchMilliSeconds();
-        BOOST_CHECK_LE(abs(actualMS - nexpectedMS), 1);
+
+        // Only compare the time when usleep() succeeds.
```

Comment:
usleep() can be interrupted by signal. When this happens, usleep() return -1. Because we are not testing usleep(), there is no need to BOOST_CHECK on this. On normal execution, this won't occur. So to make the test simple, no need to handle the signal and retry sleep.